Add "*cupsFilter" line to accept PDF input data to the PPDs, see:

  <https://www.linuxfoundation.org/collaborate/workgroups/openprinting/pdfasstandardprintjobformat>

NB, this is done in debian/rules, so it will automatically apply to
    every new PPD which gets added to this package in the future.  The
    patch should however not be deleted, so it does show up when
    looking at the differences with upstream.
--- foo2zjs_20100728dfsg.orig/PPD/Generic-OAKT_Printer.ppd
+++ foo2zjs_20100728dfsg/PPD/Generic-OAKT_Printer.ppd
@@ -33,7 +33,8 @@
 *cupsVersion:	1.0
 *cupsManualCopies: True
 *cupsModelNumber:  2
-*cupsFilter:	"application/vnd.cups-postscript 0 foomatic-rip"
+*cupsFilter: "application/vnd.cups-postscript 100 foomatic-rip"
+*cupsFilter: "application/vnd.cups-pdf 0 foomatic-rip"
 *%pprRIP:        foomatic-rip other
 *ModelName:     "Generic OAKT Printer"
 *ShortNickName: "Generic OAKT Printer foo2oak"
--- foo2zjs_20100728dfsg.orig/PPD/Generic-ZjStream_Printer.ppd
+++ foo2zjs_20100728dfsg/PPD/Generic-ZjStream_Printer.ppd
@@ -33,7 +33,8 @@
 *cupsVersion:	1.0
 *cupsManualCopies: True
 *cupsModelNumber:  2
-*cupsFilter:	"application/vnd.cups-postscript 0 foomatic-rip"
+*cupsFilter: "application/vnd.cups-postscript 100 foomatic-rip"
+*cupsFilter: "application/vnd.cups-pdf 0 foomatic-rip"
 *%pprRIP:        foomatic-rip other
 *ModelName:     "Generic ZjStream Printer"
 *ShortNickName: "Generic ZjStre. Printer foo2zjs"
